`check-types` in the browser extension breaks whenever a file is added to `public/` until you re-run `bun install`

**Severity:** Low (developer friction)
**File:** `apps/browser-extension/package.json:12`

## Description

WXT generates the `PublicPath` union that types `browser.runtime.getURL()` into
the gitignored `.wxt/types/paths.d.ts`. That generation is wired **only** to
`postinstall`:

```json
"check-types": "bun run compile",
"compile": "tsc --noEmit",
"postinstall": "wxt prepare"
```

So `.wxt/` goes stale the moment `public/` changes without a reinstall, and the
type-check fails with errors that point at correct code:

```
utils/ui-components.ts(120,43): error TS2769: No overload matches this call.
  Argument of type '"/new_logo.png"' is not assignable to parameter of type 'PublicPath'.
```

`public/new_logo.png` exists and is declared in `wxt.config.ts`
`web_accessible_resources` — the union in `.wxt/types/paths.d.ts` was simply
generated before the file was added. Running `bunx wxt prepare` regenerates it
and `tsc --noEmit` goes clean immediately, which confirms the diagnosis.

CI is unaffected (it installs before checking), so this only costs local dev time
— but the failure mode is confusing because it blames source files that are fine.

## Suggested fix

Make the task self-sufficient:

```json
"check-types": "wxt prepare && tsc --noEmit"
```

This mirrors `supermemory-mcp`, whose `check-types` already runs its
`build:widget` prerequisite for the same reason.

--- a/apps/browser-extension/package.json
+++ b/apps/browser-extension/package.json
@@ -9,10 +9,9 @@
 		"dev:firefox": "wxt -b firefox",
 		"build": "wxt build",
 		"build:firefox": "wxt build -b firefox",
-		"check-types": "bun run compile",
+		"check-types": "wxt prepare && tsc --noEmit",
 		"zip": "wxt zip",
 		"zip:firefox": "wxt zip -b firefox",
-		"compile": "tsc --noEmit",
 		"postinstall": "wxt prepare"
 	},
 	"dependencies": {

